packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Introduce `Navigation` component as `__experimentalNavigation` for displaying a hierarchy of items.
```

Comment:
Whoops, experimental features [shouldn't appear in the CHANGELOG](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis):

> To Project Contributors …
> these APIs should neither be documented nor mentioned in any CHANGELOG.

Reply:
I don't think this was correctly processed because it's under the `Unreleased` "version" but doesn't have a header like "Breaking Change" or "New Feature". This appears to have been released in `10.2.0` but has been stuck here…

cc: @gziolo

Reply:
Yes, I think there is a bug in the script that processes changelogs. It's primitive so I'm not surprised 😂

In this case you are correct that it should not be listed although as far as I remember React includes experimental APIs as well. It's all subjective.

Reply:
I opened https://github.com/WordPress/gutenberg/pull/27436 that fixes the bug you discovered.

Reply:
Thanks for taking care of this, I'll be more careful next time!